mautic/mautic
# Fix email send with company tokens

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅
| New feature/enhancement? (use the a.x branch)      | ❌
| Deprecations?                          | ❌
| BC breaks? (use the c.x branch)        | ❌
| Automated tests included?              | ✅ 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create company Test1 with address 1 as Test1
3. Create contact and assign company Test1
4. Create company Test2 with address 1 as Test2
5. Then assign company Test2 to contact as primary but let also Test1 as secondary company
6. Create email with tokens like below:
```
{contactfield=email}
{contactfield=companyname}
{contactfield=companyaddress1}
```
7. Send template email via campaign or segment email to the contact created above
8. Observe the company name and address in the email, it should be Test2
